src/Frontend/Panes/Lobbies/CreatePlanetPane.tsx

```diff
@@ -1,20 +1,26 @@
-import React, { useState } from 'react';
+import { AdminPlanet } from '@darkforest_eth/types';
+import _ from 'lodash';
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import { LobbyAdminTools } from '../../../Backend/Utils/LobbyAdminTools';
+import { Btn } from '../../Components/Btn';
 import {
   Checkbox,
   DarkForestCheckbox,
   DarkForestNumberInput,
-  NumberInput,
+  NumberInput
 } from '../../Components/Input';
+import { LoadingSpinner } from '../../Components/LoadingSpinner';
 import { Row } from '../../Components/Row';
-import { Green, Red, Sub, Subber } from '../../Components/Text';
-import { Btn } from '../../Components/Btn';
+import { Green, Red, Sub } from '../../Components/Text';
 import { Table } from '../../Views/Table';
-
 import { LobbiesPaneProps, Warning } from './LobbiesUtils';
-import styled from 'styled-components';
-import { AdminPlanet } from '@darkforest_eth/types';
-import _ from 'lodash';
+import { InvalidConfigError, toInitializers } from './Reducer';
 
+const jcFlexEnd = { display: 'flex', justifyContent: 'flex-end' } as CSSStyleDeclaration &
+  React.CSSProperties;
+const jcSpaceEvenly = { display: 'flex', justifyContent: 'space-evenly' } as CSSStyleDeclaration &
+  React.CSSProperties;
 const rowChunkSize = 4;
 const rowStyle = { gap: '8px' } as CSSStyleDeclaration & React.CSSProperties;
 // Handling the non-input lvl 0 by calculating the items in the row
```

Comment:
Do we want to allow Level 0 as input? Might make our Min planet level easier.

Reply:
yes